### PR TITLE
add information for secure docker connections

### PR DESCRIPTION
**Pull request by patrickjahns**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,16 @@ You are not required to set `DOCKER_HOST` globally. You may choose to configure
 
 If DOCKER_HOST is not set and the value is also not configured in the plugin config page, it defaults to `unix:///var/run/docker.sock`
 
+If you are working with remote docker containers, it is adviced to use [Dockers TLS security options](http://docs.docker.com/articles/https/). For client setup please see the [official documentation](http://docs.docker.com/articles/https/#client-modes)
+
+Example
+```
+DOCKER_HOST="tcp://127.0.0.1:4243"
+DOCKER_TLS_VERIFY=1
```
Comment (knownasilya) [inline README.md]: Do these need `export` as well?

File: README.md [hunk 2]
```diff
@@ -31,6 +31,16 @@ You are not required to set `DOCKER_HOST` globally. You may choose to configure
...
+export DOCKER_CERT_PATH=~/stridercd/certs
+```
+Be sure to [enable TLS on the Daemon](http://docs.docker.com/articles/https/#daemon-modes) as well. More inforation at http://docs.docker.com/articles/https/
+
```
Comment (knownasilya) [inline README.md]: information is miss-spelled

File: README.md [hunk 3]
```diff
@@ -31,6 +31,16 @@ You are not required to set `DOCKER_HOST` globally. You may choose to configure
 
 If DOCKER_HOST is not set and the value is also not configured in the plugin config page, it defaults to `unix:///var/run/docker.sock`
 
+If you are working with remote docker containers, it is adviced to use [Dockers TLS security options](http://docs.docker.com/articles/https/). For client setup please see the [official documentation](http://docs.docker.com/articles/https/#client-modes)
```
Comment (knownasilya) [inline README.md]: advised is miss-spelled

File: README.md (hunk 1 above)
Comment (patrickjahns) [inline README.md]: What do you mean by that? Both are just environment variables as well

Comment (knownasilya) [inline README.md]: But you `export` the third one, not sure why that one and not the others.

Comment (patrickjahns) [inline README.md]: Oh sorry - just saw the 2 without export and assumed you were asking if this needs export in general. I am really sorry for that. Now it is fine..

**Comment by knownasilya** [comment]
No problem :+1: One last thing, could you squash the commits? Thanks!

**Comment by patrickjahns** [comment]
Done

**Comment by patrickjahns** [comment]
Wrong button - github webinterface and me are not friends the past 2 days

**Comment by knownasilya** [comment]
@patrickjahns I know what you mean. I've tweeted at github many times to move that close button to the left..

**Comment by knownasilya** [comment]
Thanks!

**Pull request merged**